core-bundle/contao/dca/tl_content.php

```diff
@@ -153,7 +152,7 @@
 		(
 			'filter'                  => true,
 			'inputType'               => 'select',
-			'options_callback'        => array('tl_content', 'getContentElements'),
+			'options'                 => $GLOBALS['TL_CTE'],
```

Comment:
```suggestion
```
This can be removed as you do not need to have both `options` and `options_callback` (the latter always wins).

Reply:
that's the point. With this, there still is a default in the DCA, but the user permission will override it.

Reply:
Not sure I understand - that fallback will never come into effect?

Reply:
But nothing will access `$GLOBALS['TL_DCA']['fields'][…]['options']` directly. We never define `options` and `options_callback` at the same time - I just find it odd to do so here.

Reply:
In addition, the default should be `[]` (deny all if nothing is allowed) instead of `$GLOBALS['TL_CTE']` (allow all if nothing is denied), shouldn‘t it?

Reply:
But it would not deny or allow anything. The widget will never use it.

Reply:
> it will, if you only load the DCA (Controller::loadDataContainer) but don't execute the onload_callback (DC_Table::construct()).

What about that?

Reply:
See [my comment](https://github.com/contao/contao/pull/7516#discussion_r1754258875)

The widget will _always_ retrieve the value from the `options_callback`. See here: https://github.com/contao/contao/blob/6f8832ffea3c8454c90d0b09429c5654232afe39/core-bundle/contao/library/Contao/Widget.php#L1306-L1315

Reply:
Okay, then we really don‘t need this line 👍